akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedTestKit.java

```diff
@@ -170,8 +237,10 @@ public static <S, E, ES extends EventSourcedEntity<S, E>> EventSourcedTestKit<S,
    * @param func A function from EventSourcedEntity to EventSourcedEntity.Effect.
    * @return a EventSourcedResult
    * @param <R> The type of reply that is expected from invoking a command handler
+   * @deprecated Use "method(MyEntity::myCommandHandler).invoke()" instead
    */
-  public <R> EventSourcedResult<R> call(Function<ES, EventSourcedEntity.Effect<R>> func) {
+  @Deprecated(since = "3.2.0", forRemoval = true)
```

Comment:
let's bump this to the right version when we know when it will be released
3.2.0 is already out

Reply:
bumped to 3.2.1